src/libraries/Microsoft.PowerFx.Interpreter/Functions/LibraryText.cs

```diff
@@ -1196,5 +1199,224 @@ internal static bool TryGetInt(FormulaValue value, out int outputValue)
             outputValue = (int)inputValue;
             return true;
         }
+
+#pragma warning disable SA1313 // Parameter '_' should begin with a lower-case letter
+
+        internal static FormulaValue Json(EvalVisitor runner, EvalVisitorContext _, IRContext irContext, FormulaValue[] args)
+        {
+            return new JsonProcessing(runner.TimeZoneInfo, irContext, args).Process();
+        }
+
+#pragma warning restore SA1313
+
+        internal class JsonProcessing
+        {
+            private JsonFlags _flags;
+            private readonly FormulaValue[] _args;
+            private readonly IRContext _irContext;
+            private readonly TimeZoneInfo _timeZoneInfo;
+
+            internal JsonProcessing(TimeZoneInfo timezoneInfo, IRContext irContext, FormulaValue[] args)
+            {
+                _args = args;
+                _irContext = irContext;
+                _timeZoneInfo = timezoneInfo;
+            }
+
+            internal FormulaValue Process()
+            {
+                GetFlags();
+
+                if (_flags == null)
+                {
+                    return CommonErrors.GenericInvalidArgument(_irContext);
+                }
+
+                string json = JsonInternal(_args[0], 0);
+
+                if (json == null)
+                {
+                    return FormulaValue.NewBlank();

```

Comment:
>return FormulaValue.NewBlank(); [](http://example.com/codeflow?start=20&length=31)

JSON should never return a blank value - always a string (or an error)

Reply:
PA client returns "null" so I'll fix it with "null"

Reply:
But JsonInternal will never return null, so this is dead code.